src/main/java/com/marek/onlinebookstore/controller/BookController.java

```diff
@@ -66,15 +66,15 @@ public BookDto update(@PathVariable Long id, @RequestBody @Valid
 
     @ResponseStatus(HttpStatus.NO_CONTENT)
     @DeleteMapping("/{id}")
-    @PreAuthorize("hasRole('ROLE_ADMIN')")
+    @PreAuthorize("hasAnyRole('ROLE_ADMIN', 'ROLE_USER')")
```

Comment:
This will allow users with role `ROLE_USER` delete all of the books in your application.